pylint_extra_requirements.txt

```diff
@@ -1,3 +1,5 @@
 # SPDX-License-Identifier: MIT
 
 # Write extra requirements for running pylint here:
+mock
+pytest
```

Comment:
Is this change related to the integration testing?

Reply:
Yes. The file tests/integration/test_ethernet.py uses pytest and mock to make integration tests directly using python instead of using an ansible playbook. The playbook is added just so that the test harness can execute these tests automatically. 
The pylint requirements are necessary so as to not get an import error when running `tox`.